public/static/docs/command-reference/unprotect.md

```diff
@@ -1,7 +1,7 @@
 # unprotect
 
-Unprotect tracked files or directories (when the <abbr>cache</abbr> protected
-mode has been enabled with `dvc config cache`).
+Unprotect tracked files or directories (when hardlinks or symlinks have been
+enabled with `dvc config cache.type`).
```

Comment:
Maybe the -h output of `dvc unprotect` should also match this? Currently it's just "Unprotect data files or directories" but it could be changed in iterative/dvc/pull/3472